app/controllers/api/regions_controller.rb

```diff
@@ -1,4 +1,23 @@
 module Api
   class RegionsController < BaseController
+    INVALID_REGIONS_ATTRS = %w(id created_at updated_at).freeze
+
+    def edit_resource(type, id, data)
+      bad_attrs = data_includes_invalid_attrs(data)
+
+      if bad_attrs.present?
+        msg = "Attributes #{bad_attrs} should not be specified for updating a region resource"
+        raise BadRequestError, msg
+      end
```

Comment:
id/href attrs are skipped in the generic.rb's edit_resource, but I wonder if the created_at/updated_at check and and the raising of that BadRequestError should be done in there since it seems general enough.

Reply:
I'll remove the `id` check then and leave the others for now. I'll a separate PR for `created_at` and `updated_at` and see what the consensus is.

Reply:
Thanks for adding the ID_ATTRS to the list here, safer. 👍